test/EmitVerilog/rtl-dialect.mlir

```diff
@@ -7,87 +7,122 @@ module {
 
   // CHECK-LABEL: // external module E
 
-  rtl.module @TESTSIMPLE(%a: i4, %cond: i1) -> (%r0: i4, %r1: i4, %r2: i4, %r3: i4, %r4: i4, %r5: i4, %r6: i4, %r7: i4, %r8: i4, %r9: i4,
-%r10: i1, %r11: i1, %r12: i1, %r13: i1, %r14: i1, %r15: i1, %r16: i1, %r17: i1, %r18: i1, %r19: i1,
-%r20: i1, %r21: i1, %r22: i1, %r23: i12, %r24: i2, %r25: i9, %r26: i9, %r27: i4
-  ) {
-    %0 = rtl.add %a, %a : i4
-    %1 = rtl.sub %a, %a : i4
-    %2 = rtl.mul %a, %a : i4
-    %3 = rtl.div %a, %a : i4
-    %4 = rtl.mod %a, %a : i4
-    %5 = rtl.shl %a, %a : i4
-    %6 = rtl.shr %a, %a : i4
-    %7 = rtl.or %a, %a : i4
-    %8 = rtl.and %a, %a : i4
-    %9 = rtl.xor %a, %a : i4
-    %10 = rtl.icmp "eq" %a, %a : i4
-    %11 = rtl.icmp "ne" %a, %a : i4
-    %12 = rtl.icmp "slt" %a, %a : i4
-    %13 = rtl.icmp "sle" %a, %a : i4
-    %14 = rtl.icmp "sgt" %a, %a : i4
-    %15 = rtl.icmp "sge" %a, %a : i4
-    %16 = rtl.icmp "ult" %a, %a : i4
-    %17 = rtl.icmp "ule" %a, %a : i4
-    %18 = rtl.icmp "ugt" %a, %a : i4
-    %19 = rtl.icmp "uge" %a, %a : i4
-    %20 = rtl.andr %a : i4
-    %21 = rtl.orr %a : i4
-    %22 = rtl.xorr %a : i4
-    %23 = rtl.concat %a, %a, %a : (i4, i4, i4) -> i12
-    %24 = rtl.extract %a from 1 : (i4) -> i2
-    %25 = rtl.sext %a : (i4) -> i9
-    %26 = rtl.zext %a : (i4) -> i9
-    %27 = rtl.mux %cond, %a, %a : i4
+  rtl.module @TESTSIMPLE(%a: i4, %b: i4, %cond: i1) -> (
+    %r0: i4,
```

Comment:
Ok if you prefer it this way, but this seems like egregious flattening of the argument list :-) 

I'd wrap to a couple lines instead of 30.